w1thermsensor/core.py

```diff
@@ -101,14 +103,16 @@ def __init__(self, sensor_type=None, sensor_id=None):
             :raises KernelModuleLoadError: if the w1 therm kernel modules could not be loaded correctly
             :raises NoSensorFoundError: if the sensor with the given type and/or id does not exist or is not connected
         """
+        super(W1ThermSensor, self).__init__()
```

Comment:
I don't think it's needed here. The mro is absolutely fine without this line - even with multiple inheritance.